src/index.js

```diff
@@ -0,0 +1,162 @@
+'use strict';
+
+function Modal(clickElm, modalParent, options) {
+  options = options || {};
+
+  var that = this;
+  clickElm.addEventListener('click', function(e) {
+    e.preventDefault();
+    that.show();
+  }, false);
+  this.clickElm = clickElm;
```

Comment:
It would be a nice interface if you could also do ``client = PaymentClient('#product-123')`` instead of pass in an actual element. Is that feasible to implement? I know newer agents have ``document.querySelector()`` but we'd need a fallback.